_commands/basics/echo.md

```diff
@@ -0,0 +1,68 @@
+---
+---
+
+echo
+--
+
+`echo` is used to output whatever it is given into stdio. Often useful see the value of enviromental variables.
+
+~~~ bash
+$ echo "Hello World!"
+Hello World!
+$ echo $USER
+<your username>
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `echo -e [string...]`
+
+<pre>
+$ echo "Hello \tWorld!\n"
+Hello \tWorld\n 
+$ echo -e "Hello \tWorld!\n"
+Hello   World 
+
+$ echo -e "\033[31mHello \033[36mWorld\033[m"  
+<span style="color:red;">Hello</span><span style="color:cyan;"> World</span>
+$ echo "\033[31mHello \033[36mWorld\033[m"
+\033[31mHello \033[36mWorld\033[m
+</pre>
+
+##### Break it down
+ * The `-e` flag will have echo evaluate the expression inside the string and used
```

Comment:
`-e` does not cause echo to evaluate expressions, it only affects escape characters